Musical Collection/Musical Collection Console App/Classes/Artist.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Musical_Collection_Console_App.Interfaces;
+using Musical_Collection_Console_App.Interfaces.Classes_Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Musical_Collection_Console_App.Classes
+{
+    public class Artist : User, IArtist
+    {
+
+        private List<string> _songsNames;
+        private List<string> _albumNames;
+
+        public Artist(string username, string password, string fullName, string birthDate) : base(username, password, fullName, birthDate)
```

Comment:
You can split this line 
Ex : 
Artist(string username, string password, string fullName, string birthDate) 
         : base(username, password, fullName, birthDate)